Microsoft.Toolkit.Win32.UI.Controls/Interop/WinRT/CoreAcceleratorKeyEventType.cs

```diff
@@ -0,0 +1,27 @@
+// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+namespace Microsoft.Toolkit.Win32.UI.Controls.Interop.WinRT
+{
+    public enum CoreAcceleratorKeyEventType
```

Comment:
@normesta @mcleanbyron In general, what do you want to do about types that are already part of WinRT that I had to duplicate to avoid the requirement of linking the Windows.winmd file? They're duplicated to allow developers on down-level/pre-RS4 machines to still perform designer work without invoking the types and getting errors because the platform doesn't support them. I've been doing something like
```
    /// <summary>Defines errors encountered during operations involving web services, such as authentication, proxy configuration, and destination URIs.</summary>
    /// <remarks>Copy from <see cref="Windows.Web.WebErrorStatus"/> to avoid requirement to link Windows.winmd</remarks>
    /// <seealso cref="Windows.Web.WebErrorStatus"/>
```
and 
```
    /// <summary>
    /// Provides data for the <see cref="IWebView.MoveFocusRequested"/> event. This class cannot be inherited.
    /// </summary>
    /// <remarks>Copy from <see cref="Windows.Web.UI.Interop.WebViewControlMoveFocusRequestedEventArgs"/> to avoid requirement to link Windows.winmd</remarks>
    /// <seealso cref="EventArgs" />
    /// <seealso cref="Windows.Web.UI.Interop.WebViewControlMoveFocusRequestedEventArgs"/>
```

A lot more documentation updates are in the `rjmurillo/WebView_Controls_WIP` branch